pyanaconda/modules/network/network.py

```diff
@@ -306,6 +313,35 @@ def is_connecting(self):
             log.debug("Connectivity state can't be determined, assuming not connecting.")
             return False
 
+    @property
+    def capabilities(self):
+        """Capabilities of the network backend."""
+        if not self.nm_available:
+            log.debug("Capabilities can't be determined.")
+            return []
+
+        return self._capabilities
+
+    def set_capabilities(self, capabilities):
+        """Set network capabilities."""
+        self._capabilities = capabilities
+        self.capabilities_changed.emit()
+        self.module_properties_changed.emit()
+        log.debug("Capabilities: %s", capabilities)
+
+    @staticmethod
```

Comment:
Why static?

Reply:
Because it is kind of helper function independent of the instance.

Reply:
Alright, I am just used to specify function at static when I need to utilize what the static method offers (static attributes  etc)
In this case, it does not provide any benefit, from what I can see. It's just more code, but if you follow this pattern in the backend, I also dont insist.